quilt/test/build.yml

```diff
@@ -1,7 +1,11 @@
 ---
 contents:
   dataframes:
-    csv: [csv, data/10KRows13Cols.csv]
-    tsv: [tsv, data/10KRows13Cols.tsv]
-    xls: [xlsx, data/10KRows13Cols.xlsx]
-  README: [raw, data/README.md]
+    csv: 
+      file: data/10KRows13Cols.csv
```

Comment:
The fact that the dataframe has the same name as the transform might actually confuse people. (And really, it should be the name of the contents.)

As for omitting the transform... I guess I can see both sides of the argument, but I kinda like having to explicitly set it.